### ISSUE TITLE
support docker compose versions >2.0

### ISSUE THREAD

**Issue by grexe** [testcontainers/testcontainers-java#422]
seems versions > 2.0 are not supported.
E.g. having a correct version `2.1`, `2.2` or even the latest `3.x` results in an error message:
```
[docker/compose:1.8.0] - STDERR: Version in `<compose.yaml>` is unsupported.
```
This is because the docker/compose container used is over a year old, latest is 1.15, see:
https://store.docker.com/community/images/docker/compose/tags

Please update the docker container used for compose containers so recent YAML files are supported.
Sadly this cannot be overridden like for other containers mentioned in [custom containers](https://www.testcontainers.org/usage.html#using-custom-utility-images)

**Comment by grexe** [comment]
the 1.8.0 version is hard coded into `ContainerisedDockerCompose`, would be easy to make it parametrized via custom containers properties if there are no assumptions/dependencies in the code regarding the compose version:
```
    public ContainerisedDockerCompose(List<File> composeFiles, String identifier) {
        super("docker/compose:1.8.0");
```
I'm working on a PR to fix this, ok?

**Comment by bsideup** [comment]
Hey @grexe,
PR sounds great, looking forward to it! 👍

**Comment by grexe** [comment]
looks quite good here, just testing with my app (the original goal,-)

**Comment by grexe** [comment]
fixed in grexe/testcontainers/#425
